developer/src/kmc-kmn/src/compiler/kmn-compiler-messages.ts

```diff
@@ -79,649 +79,644 @@ export class KmnCompilerMessages {
 
-  static Error_CannotReadInfile                               = () => m(this.ERROR_CannotReadInfile, `Cannot read the input file`);
   static ERROR_CannotReadInfile                               = SevError | 0x008; // #10678: reduced from fatal to error in 17.0
+  static Error_CannotReadInfile                               = () => m(this.ERROR_CannotReadInfile, `Cannot read the input file`);
 
-  static Fatal_SomewhereIGotItWrong                           = () => m(this.FATAL_SomewhereIGotItWrong, `Internal error: contact Keyman`);
   static FATAL_SomewhereIGotItWrong                           = SevFatal | 0x009;
+  static Fatal_SomewhereIGotItWrong                           = () => m(this.FATAL_SomewhereIGotItWrong, `Internal error: contact Keyman`);
 
-  static Error_InvalidToken                                  = () => m(this.ERROR_InvalidToken, `Invalid token found`);
   static ERROR_InvalidToken                                   = SevError | 0x00A;
+  static Error_InvalidToken                                   = () => m(this.ERROR_InvalidToken, `Invalid token found`);
 
   // kmcmplib: static Error_InvalidBegin                                  = () => m(this.ERROR_InvalidBegin, `Invalid 'begin' command`);
+  static ERROR_InvalidBegin                                   = SevError | 0x00B;
   static Error_InvalidBegin = () => mw(this.ERROR_InvalidBegin,
     `A "begin unicode" statement is required to compile a KeymanWeb keyboard`);
-  static ERROR_InvalidBegin                                   = SevError | 0x00B;
 
-  static Error_InvalidName                                   = () => m(this.ERROR_InvalidName, `Invalid 'name' command`);
   static ERROR_InvalidName                                    = SevError | 0x00C;
+  static Error_InvalidName                                    = () => m(this.ERROR_InvalidName, `Invalid 'name' command`);
 
-  static Error_InvalidVersion                                = () => m(this.ERROR_InvalidVersion, `Invalid 'version' command`);
   static ERROR_InvalidVersion                                 = SevError | 0x00D;
+  static Error_InvalidVersion                                 = () => m(this.ERROR_InvalidVersion, `Invalid 'version' command`);
 
-  static Error_InvalidLanguageLine                           = () => m(this.ERROR_InvalidLanguageLine, `Invalid 'language' command`);
   static ERROR_InvalidLanguageLine                            = SevError | 0x00E;
+  static Error_InvalidLanguageLine                            = () => m(this.ERROR_InvalidLanguageLine, `Invalid 'language' command`);
 
-  static Error_LayoutButNoLanguage                           = () => m(this.ERROR_LayoutButNoLanguage, `Layout command found but no language command`);
   static ERROR_LayoutButNoLanguage                            = SevError | 0x00F;
+  static Error_LayoutButNoLanguage                            = () => m(this.ERROR_LayoutButNoLanguage, `Layout command found but no language command`);
```

Comment:
Are commands generally in single quotes? e.g. 'language' command like l. 253?

Reply:
Yeah but not trying to change any messages in this PR!